_integration-test/run.sh

```diff
@@ -135,3 +135,10 @@ source ./custom-ca-roots/setup.sh
 $dcr --no-deps web python3 /etc/sentry/test-custom-ca-roots.py
 source ./custom-ca-roots/teardown.sh
 echo "${_endgroup}"
+
+for service in $(docker compose ps --format json | jq -r '.[]| select(.State != "running") | .Service'); do
+  # geoipupdate is not set up, so we can safely ignore it, it will fail if not configured.
+  if [[ $service != "geoipupdate" ]]; then
+    echo "ERROR: Service $service has failed after the integration tests!"
```

Comment:
I'd recommend outputting the full `docker compose ps` so we have better debugging helps, maybe even logs of the failing service.

Also do we want to exit with a non-zero status so that CI goes red?

Reply:
Done! I figured the logs may be too verbose if multiple services are failing.